packages/module/patternfly-docs/content/examples/basic.md

```diff
@@ -15,33 +15,41 @@ propComponents: ['FeedbackModalProps']
 import { FeedbackModal } from "@patternfly/react-user-feedback";
 import feedbackImage from '@patternfly/react-user-feedback/dist/esm/images/rh_feedback.svg';
 
-## Examples
-### Basic Example
+The user feedback extension is a PatternFly React based component used to collect user feedback.  Examples of how to use this extension are documented below.  This extensions allows users to submit feedback, report a bug, request support, as well as join a mailing list to stay updated on the latest news and research opportunities.
 
+## Examples
+### Basic example
+This is an example of the minimum needed to create a feedback modal.  A user can create a feedback modal that just contains an option to provide feedback.
 ```js file="./Basic.tsx"
 
 ```
 
-### URL Example
+### URL example
+Developers have the option to use URLs instead of the built in forms to collect data.  This is useful if you have a custom form that you want to use to collect data.  The URL will be opened in a new tab.
 
 ```js file="./URL.tsx"
 
 ```
 
 ### Advanced feedback modal
-
+This modal is an example of the using the built in forms to collect data.
 ```js file="./Advanced.tsx"
 
 ```
 
 ### Advanced with email passed to modal
+A developer can also pass in the email address for a user to the feedback modal.  This is useful if you have a user's email address and want to pre-populate the email field in the feedback modal.
```

Comment:
To automatically pass a user's email address into a feedback modal, use the `email` property. This allows you to pre-populate the modal's email field.

Reply:
Done.